common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -108,6 +108,8 @@
         }
     }
 
+    private static final LeakEntry LEAK_ENTRY = new LeakEntry();
```

Comment:
@Scottmitch sure let me do this... that said I don't think it makes any difference